docs/src/doc/user-guide/methods.md

```diff
@@ -13,6 +13,12 @@ class RotatingCube: Spatial() {
 }
 ```
 
+!!! danger ""
+    All methods that you register, are registered in `snake_case`!  
+    So `myVeryCoolFunction` will become `my_very_cool_function`  
```

Comment:
Let it wrap naturally, it's okay that the lines are longer. When rendered it will be of the appropriate length.

Reply:
I wasn't wrapping it because of line lenght, i was wrapping because i thought it makes sense. But I'll change it :-)